scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -41,6 +83,111 @@ namespace <%= namespace %>.SDK.iOS {
             });
         }
 
+        private static bool IsShippingAddressField(string field) {
+            return ShippingAddressFieldsSet.Contains(field);
+        }
+
+        private static bool IsContactField(string field) {
+            return ShippingContactFieldsSet.Contains(field);
+        }
+
+        private static bool IsPartialShippingAddressField(string field) {
+            return PartialShippingAddressFieldsSet.Contains(field);
+        }
+
+        private static UpdateRequestStatus GetUpdateRequestStatusFromCheckoutUserErrors(List<UserError> errors) {
+
+            var payErrors = new List<ApplePayError>();
+
+            // Check to see if any of the user errors are not billing address errors
+            // If it is not a billing address error return a failure
+            foreach (var error in errors) {
+                var fields = error.field();
+                if (fields.Contains("billingAddress")) {
+                    string description = error.message();
+                    string lastField = fields[fields.Count - 1];
```

Comment:
I don't think it would be appropriate here to throw an exception, there is no way we can handle it either. The only option is to let it 💥 or handle it gracefully by saying it's a failure. The case where the number of fields is 0 is when the SDK is broken or the Storefront API is broken.